docs/configuration/config_yaml.rst

```diff
@@ -62,6 +62,18 @@ Use the following settings to enable access control:
      # header and login the user without further checks. Use with caution.
      checkBasicAuthenticationPassword: true
 
+     # Whether to trust remote user headers. If you have setup authentication in front of
+     # OctoPrint and the user names you use there match OctoPrint accounts, by setting this to true users will
+     # be logged into OctoPrint as the user provided in the header. Your should ONLY ENABLE THIS if your
+     # OctoPrint instance is only accessible through a connection locked down through an authenitating reverse proxy!
+     trustBasicAuthentication: false
```

Comment:
There seems to be a documentation error here, this should say `trustRemoteUser`.